exporters/otlp/include/opentelemetry/exporters/otlp/otlp_http_exporter.h

```diff
@@ -65,6 +68,7 @@ struct OtlpHttpExporterOptions
 
   // TODO: Enable/disable to verify SSL certificate
   // TODO: Reuqest timeout
```

Comment:
This TODO could be removed?

Reply:
Yes. This PR is already merged, maybe it can be remove in another PR later.

Reply:
sorry if I merged it a bit earlier. There is a task to remove/handle invalid TODOs from the code base, this can be handled through that if not done before.